.gitignore

```diff
@@ -87,6 +87,12 @@ celerybeat.pid
 celerybeat-schedule
 
 # Deploy
-conf/Dockerrun.aws.json
+*Dockerrun.aws.json
 conf/nginx.ureport.conf
 conf/deploy.zip
+*.elasticbeanstalk
+
+# Elastic Beanstalk Files
+.elasticbeanstalk/*
+!.elasticbeanstalk/*.cfg.yml
+!.elasticbeanstalk/*.global.yml
```

Comment:
These are the lines that get automatically added (will remove above) - maybe they're hinting we should put the changes we want to persist in *.cfg.yml or *.glocabl.yml files.